pkg/providers/p11.go

```diff
@@ -414,6 +415,26 @@ func (p *P11) GenerateSKey(ctx context.Context, request *istio.GenerateSKeyReque
 	return
 }
 
+// ImportCACert inserts the Root CA cert chain
```

Comment:
Looks fine - have added a couple of small changes in my MR, including saving the request.KekKid as p.kid for future use